nixos/platform/network.nix

```diff
@@ -776,10 +776,15 @@ in
         command = let
           links = lib.concatStringsSep " " (map (i: i.link) fclib.underlay.links);
         in
-          "${pkgs.fc.check-link-redundancy}/bin/check_link_redundancy ${links}";
+          "/run/wrappers/bin/sudo ${pkgs.fc.check-link-redundancy}/bin/check_link_redundancy ${links}";
```

Comment:
The absolute path is not *strictly* necessary, as sudo is in the environment anyways. But it's fine.